viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -287,6 +287,7 @@ void CartoFacade::IOInit() {
             algo_config.missing_data_ray_length);
         map_builder.OverwriteMaxRange(algo_config.max_range);
         map_builder.OverwriteMinRange(algo_config.min_range);
+        map_builder.OverwriteUseIMUData(algo_config.use_imu_data);
```

Comment:
[note to self] imu enabled must be defined for the `map_builder` AND during initialization in order to register the `kIMUSensor`

Reply:
See Viam-cartographer.go lines 440-445